Nos/Views/Settings/SettingsView.swift

```diff
@@ -41,220 +42,230 @@ struct SettingsView: View {
+                    #endif
+                }
+            header: {
                 Text(.localizable.debug)
                     .foregroundColor(.primaryTxt)
                     .font(.clarity(.semibold, textStyle: .headline))
                     .textCase(nil)
                     .listRowInsets(EdgeInsets())
                     .padding(.vertical, 15)
-            }
+                }
             .listRowGradientBackground()
-            
-            ActionButton(
-                title: .localizable.deleteMyAccount,
-                font: .clarityBold(.title3),
-                padding: EdgeInsets(top: 16, leading: 0, bottom: 16, trailing: 0),
-                depthEffectColor: .actionSecondaryDepthEffect,
-                backgroundGradient: .verticalAccentSecondary,
-                shouldFillHorizontalSpace: true
-            ) {
-                alert = AlertState(
-                    title: { TextState(String(localized: .localizable.deleteAccount)) },
-                    actions: {
-                        ButtonState(role: .destructive, action: .send(.deleteAccount)) {
-                            TextState(String(localized: .localizable.delete))
+
+                ActionButton(
+                    title: .localizable.deleteMyAccount,
+                    font: .clarityBold(.title3),
+                    padding: EdgeInsets(top: 16, leading: 0, bottom: 16, trailing: 0),
+                    depthEffectColor: .actionSecondaryDepthEffect,
+                    backgroundGradient: .verticalAccentSecondary,
+                    shouldFillHorizontalSpace: true
+                ) {
+                    showDeleteConfirmationAlert = true
+                }
+                .clipShape(Capsule())
+                .listRowBackground(Color.clear)
+                .listRowInsets(EdgeInsets())
+            }
+            .scrollContentBackground(.hidden)
+            .background(Color.appBg)
+            .nosNavigationBar(title: .localizable.settings)
+            .onAppear {
+                privateKeyString = currentUser.keyPair?.nsec ?? ""
+                analytics.showedSettings()
+            }
+
+            if showDeleteConfirmationAlert {
+                /// Adds a translucent background overlay to the view's man content
+                Color.actionSheetOverlay.opacity(0.5)
+                    .ignoresSafeArea()
+
+                DeleteConfirmationView(
+                    requiredText: String(localized: "delete").uppercased(),
+                    onDelete: {
+                        Task {
+                            await alertButtonTapped(.deleteAccount)
                         }
+                        showDeleteConfirmationAlert = false
                     },
-                    message: { TextState(String(localized: .localizable.deleteAccountDescription)) }
+                    onCancel: {
+                        showDeleteConfirmationAlert = false
+                    }
                 )
```

Comment:
```suggestion
                )
                .transition(.opacity) 
                .animation(.easeInOut(duration: 0.3), value: showDeleteConfirmationAlert)
```

I didn't test this code, but I think a little fade animation would be nice when hiding or showing this view.

Reply:
added, but I didn't really notice any significant difference 🤔